analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.CSharp8.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.IO;
+
+namespace Tests.Diagnostics
+{
+    namespace CSharp8
+    {
+        public class Disposable : IDisposable
+        {
+            public void Dispose() { }
+        }
+
+        class UsingDeclaration
+        {
+            public void Disposed_UsingDeclaration()
+            {
+                using var d = new Disposable(); // Noncompliant FP
+                d.Dispose(); // FIXME Non-compliant {{Refactor this code to make sure 'd' is disposed only once.}}
```

Comment:
What does this mean and what is the plan?
We raise, but on a different location.

It changes the behavior. At the same time, is it worth changing? It says to refactor the code.

Reply:
I did not touch the using part of the UTs yet.
 
 I will mark everything related as FN and see if it will be fixed by #7339.
 
 This one - weirdly it's raising correctly. 
 I will see to change the message for the using case to make sense.